custom/icds_reports/tasks.py

```diff
@@ -603,7 +603,8 @@ def email_dashboad_team(aggregation_date):
     if six.PY2 and isinstance(aggregation_date, bytes):
         aggregation_date = aggregation_date.decode('utf-8')
     # temporary soft assert to verify it's completing
-    _dashboard_team_soft_assert(False, "Aggregation completed on {}".format(settings.SERVER_ENVIRONMENT))
+    if not settings.UNIT_TESTING:
+        _dashboard_team_soft_assert(False, "Aggregation completed on {}".format(settings.SERVER_ENVIRONMENT))
```

Comment:
This should be getting caught in tests already: https://github.com/dimagi/commcare-hq/blob/5f65beda0d2b6993bb799c6c966d8dd9ad43bdb1/custom/icds_reports/tests/__init__.py#L184 and I used it pretty frequently with REUSE_DB=1 when I was in this code.

What were the tests that were causing this error to pop up locally?

Reply:
Hmm, I haven't been able to reproduce again but definitely saw that error during testing.

Either way, it seems better to skip the soft assert during testing because raising the soft assert skips the remainder of ```email_dashboad_team``` during testing.

Reply:
Yeah that sounds good. Maybe we should remove the other catch in that case? (i'm about to have a cleanup pr that I can take that on)